content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -544,6 +544,10 @@ Then, on the node being removed, reset all kubeadm installed state:
 ```bash
 kubeadm reset
 ```
+The reset process does not reset or clean up iptables rules or IPVS tables. If you wish to reset iptables, you must do so manually.
```

Comment:
add empty line bellow the text.
also end it with `:`.

Reply:
@neolit123 Thanks for your review.
Is it like this?
```bash
kubeadm reset 
```

The reset process does not reset or clean up iptables rules or IPVS tables. If you wish to reset iptables, you must do so manually.

```bash
iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
```

Reply:
hi, yes.

Reply:
@neolit123 Thank you for the prompt reply and I already fix it.